manifests/n/Neurodesk/NeurodeskApp/1.0.0/Neurodesk.NeurodeskApp.locale.en-US.yaml

```
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Neurodesk.NeurodeskApp
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: University of Queensland

```

```suggestion
Publisher: Project Jupyter
```
This must match Control Panel.